src/extension/android/adb.ts

```diff
@@ -108,23 +108,10 @@ export class AdbHelper {
         let enableDebugCommand = `${this.adbExecutable} ${
             debugTarget ? "-s " + debugTarget : ""
         } shell am broadcast -a "${packageName}.RELOAD_APP_ACTION" --ez jsproxy ${enable}`;
-        return new CommandExecutor(this.nodeModulesRoot, projectRoot)
-            .execute(enableDebugCommand)
-            .then(() => {
-                // We should stop and start application again after RELOAD_APP_ACTION, otherwise app going to hangs up
-                return new Promise(resolve => {
-                    setTimeout(() => {
-                        this.stopApp(projectRoot, packageName, debugTarget, appIdSuffix).then(
-                            () => {
-                                return resolve(void 0);
-                            },
-                        );
-                    }, 200); // We need a little delay after broadcast command
-                });
-            })
-            .then(() => {
-                return this.launchApp(projectRoot, packageName, debugTarget, appIdSuffix);
-            });
+        await new CommandExecutor(this.nodeModulesRoot, projectRoot).execute(enableDebugCommand);
+        await PromiseUtil.delay(200); // We need a little delay after broadcast command
```

Comment:
```suggestion
        // We should stop and start application again after RELOAD_APP_ACTION, otherwise app going to hangs up
        await PromiseUtil.delay(200); // We need a little delay after broadcast command
```